app/models/user.rb

```diff
@@ -226,6 +226,10 @@ def all_groups
     miq_groups
   end
 
+  def group?(group)
```

Comment:
Seems like this could use a better name. How about 'groups_include?' I'm not known for coming up with good names so this is just a suggestion. If you have something better go for it.